chia/data_layer/data_layer_wallet.py

```diff
@@ -542,6 +578,14 @@ async def create_report_spend(self, launcher_id: bytes32) -> Tuple[TransactionRe
             type=uint32(TransactionType.OUTGOING_TX.value),
             name=singleton_record.coin_id,
         )
+        if fee > 0:
+            chia_tx = await self.create_tandem_xch_tx(fee, expected_announcement, coin_announcement=False)
+            aggregate_bundle = SpendBundle.aggregate([dl_tx.spend_bundle, chia_tx.spend_bundle])
+            dl_tx = dataclasses.replace(dl_tx, spend_bundle=aggregate_bundle)
+            chia_tx = dataclasses.replace(chia_tx, spend_bundle=None)
+            txs: List[TransactionRecord] = [dl_tx, chia_tx]
```

Comment:
This List is always either one item (dl_tx) or two items (dl_tx, chia_tx) , correct?

Reply:
Correct, but the reason it's a list is to keep consistentcy with the equivalent wallet methods `generate_signed_transaction` which should be returning lists of tx records (CAT wallet does right now, Standard Wallet does not).